dev/integration_tests/flutter_gallery/pubspec.yaml

```diff
@@ -35,19 +35,19 @@ dependencies:
   plugin_platform_interface: 2.1.8 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   source_span: 1.10.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   term_glyph: 1.2.1 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  url_launcher_android: 6.2.2 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  url_launcher_ios: 6.2.4 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  url_launcher_android: 6.3.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
It looks like picking up https://github.com/flutter/packages/pull/5708 is breaking our android tests:

```
[        ] Execution failed for task ':url_launcher_android:generateDebugRFile'.
[        ] > Could not resolve all files for configuration ':url_launcher_android:debugCompileClasspath'.
[        ]    > Could not resolve androidx.annotation:annotation:1.7.1.
[        ]      Required by:
[        ]          project :url_launcher_android
[        ]       > Cannot find a version of 'androidx.annotation:annotation' that satisfies the version constraints:
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'androidx.annotation:annotation:1.7.1'
[        ]            Constraint path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'androidx.annotation:annotation:{strictly 1.7.0}' because of the following reason: dependency was locked to version '1.7.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'androidx.core:core:1.10.1' (releaseVariantReleaseApiPublication) --> 'androidx.annotation:annotation:1.6.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'androidx.browser:browser:1.5.0' (releaseVariantReleaseApiPublication) --> 'androidx.annotation:annotation:1.2.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.lifecycle:lifecycle-common:2.3.1' (apiElements) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.lifecycle:lifecycle-common-java8:2.2.0' (compile) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.lifecycle:lifecycle-runtime:2.3.1' (releaseApiPublication) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'androidx.core:core:1.10.1' (releaseVariantReleaseApiPublication) --> 'androidx.versionedparcelable:versionedparcelable:1.1.1' (compile) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.lifecycle:lifecycle-runtime:2.3.1' (releaseApiPublication) --> 'androidx.arch.core:core-common:2.1.0' (compile) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.collection:collection:1.1.0' (compile) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.viewpager:viewpager:1.0.0' (compile) --> 'androidx.annotation:annotation:1.0.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.loader:loader:1.0.0' (compile) --> 'androidx.annotation:annotation:1.0.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.activity:activity:1.0.0' (compile) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.lifecycle:lifecycle-viewmodel:2.1.0' (compile) --> 'androidx.annotation:annotation:1.1.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.viewpager:viewpager:1.0.0' (compile) --> 'androidx.customview:customview:1.0.0' (compile) --> 'androidx.annotation:annotation:1.0.0'
[        ]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.activity:activity:1.0.0' (compile) --> 'androidx.savedstate:savedstate:1.0.0' (compile) --> 'androidx.annotation:annotation:1.1.0'
[  +94 ms]            Dependency path 'io.flutter.plugins.urllauncher:url_launcher_android:1.0-SNAPSHOT' --> 'io.flutter:flutter_embedding_debug:1.0.0-6cefbe183546461bdcdf7633a4cd5021cce2f210' (compile) --> 'androidx.fragment:fragment:1.1.0' (compile) --> 'androidx.loader:loader:1.0.0' (compile) --> 'androidx.lifecycle:lifecycle-livedata:2.0.0' (compile) --> 'androidx.arch.core:core-runtime:2.1.0' (compile) --> 'androidx.annotation:annotation:1.1.0'
```

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8753652060239479265/+/u/run_test.dart_for_build_tests_shard_and_subshard_3_3/stdout

cc @gmackall

Reply:
Looks like it is another lockfile issue, the build succeeds without them. Let me try regenerating the lockfiles and see if I can get it into a passing state